Gems/Atom/Feature/Common/Assets/Shaders/Reflections/ReflectionScreenSpaceBlurVertical.azsl

```diff
@@ -20,7 +20,8 @@ ShaderResourceGroup PassSrg : SRG_PerPass
     float m_invOutputScale;
 }
 
-#include <Atom/RPI/ShaderResourceGroups/DefaultDrawSrg.azsli>
+// The shader variant system needs a fallback, using UnusedFallbackDrawSrg for this.
```

Comment:
What error do you get if you simply remove DefaultDrawSrg.azsli?

Reply:
Generally speaking, you'll see something like

4/4/2023 9:27 AM | ERROR | AZSLc: D:\GitHub\tinyprbranch\o3de\AutomatedTesting\user\AssetProcessorTemp\JobTemp-VmDfMW\LightCullingTilePrepare_dx12.azslin(,) : IR error #129: If you have non-static options, one SRG must be designated as the default ShaderVariantFallback from AssetBuilder

Although in this particular case, the original shader did not need to include the DefaultDrawSrg in the first place, because it does not use any shader variants, so this one can just be removed.

Reply:
yeah. My original confusion is why do we need to include an azsli file  whose SRG is not even used in the shader.

Reply:
Yeah, even if the Srg is not used, we still need a fallback for the shader variant system (some shaders use SRG_PerPass_WithFallback for this purpose, others use SRG_PerDraw). But in this case, it also does not use shader options, so it is not necessary.